docker/Dockerfile

```diff
@@ -60,9 +60,7 @@ FROM ubuntu:jammy AS stage_deploy
 
 COPY --from=stage_build /emsdk /emsdk
 
-# Fallback in case Emscripten isn't activated.
-# This will let use tools offered by this image inside other Docker images
-# (sub-stages) or with custom / no entrypoint
+# Set up emsdk environment
 ENV EMSDK=/emsdk \
     EMSDK_NODE=/emsdk/node/15.14.0_64bit/bin/node \
```

Comment:
Seeing this now I think we should remove `EMSDK_NODE` here.. its not used for anything that I know of.

Reply:
Also, EMSDK it only used locally within this file.. is there some way to declare it as local?